lib/Subscribers/ConfirmationEmailMailer.php

```diff
@@ -29,6 +31,12 @@ function sendConfirmationEmail(Subscriber $subscriber) {
       return false;
     }
 
+    $subscriber->count_confirmations++;
+    $subscriber->save();
+    if($subscriber->count_confirmations > self::MAX_CONFIRMATION_EMAILS) {
```

Comment:
according to spec: `Do not enforce this limit for logged-in users.` How is this done?

Do you think this functionality needs a test?

Reply:
Fixed, I must have removed it sometime during testing. 

I was thinking about tests, but didn't find real usecase for them. Problem would be, if emails weren't sent at all, but there is test for that. Subscription itself is continuing anyway, so it's just about how many emails user receives. But I don't have strong opinion about that. If you think there should be test, I can add it.

Reply:
I think there definitely should be a test. Your condition is a bit complex so it deserves one test. And this class is using a DI so it shouldn't be that difficult to add tests for different use-cases.